app/services/dependency-resolver.js

```diff
@@ -43,34 +44,50 @@ export default Ember.Service.extend({
   },
 
   resolveAddons: function(addons, dependencies) {
-    let addonPromises = {};
-    Object.keys(addons).forEach((name) => {
-      var value = addons[name];
-
-      addonPromises[name] = this.resolveAddon(name, value);
+    const taskInstance = this.get('resolveAddonsTask').perform(addons, dependencies);
+    return taskInstance.then(() => {
+      return RSVP.resolve(task.value);
```

Comment:
`task` is an import (from `ember-concurrency`), and is a method AFAIK (not a thing with a `.value` property)